test/methods.js

```diff
@@ -760,7 +760,7 @@ test( "bic", function() {
 	ok( !method( "PBNKDEFFXXX1" ), "Invalid BIC: too long" );
 	ok( !method( "1BNKDEFF" ), "Invalid BIC: invalid digit" );
 	ok( !method( "PBNKDE1F" ), "Invalid BIC: invalid digit" );
-	ok( !method( "PBNKDEF3" ), "Invalid BIC: invalid digit" );
+	ok( !method( "PBNKDEF0" ), "Invalid BIC: invalid digit" );
```

Comment:
Next line tests for letter "O", but testing for digit zero "0" is done few lines lower, so i will remove this, now redundant line.